ArduPlane/Attitude.cpp

```diff
@@ -529,7 +529,12 @@ void Plane::calc_nav_pitch()
  */
 void Plane::calc_nav_roll()
 {
-    nav_roll_cd = constrain_int32(nav_controller->nav_roll_cd(), -roll_limit_cd, roll_limit_cd);
+    //Have we received a msg that guides roll in the last 3 seconds?
+    //  If so, don't updated nav_roll_cd as it is being guided externally.
+    if( ! plane.guided_state.guiding_roll ||
+          AP_HAL::millis() - plane.guided_state.last_guided_ms > 3000) {
+        nav_roll_cd = constrain_int32(nav_controller->nav_roll_cd(), -roll_limit_cd, roll_limit_cd);
```

Comment:
I don't think it's wise to remove this roll restriction, it's there for the safety of the aircraft and controlled by params. If they need to be bigger in guided then they are probably ok in AUTO too so the param should change. Can you give a use-case for this?